tools/api-markdown-documenter/src/api-item-transforms/ApiItemUtilities.ts

```diff
@@ -512,10 +512,22 @@ export function doesItemKindRequireOwnDocument(
 /**
  * Determines whether or not the specified API item is one that should be rendered to its own document.
  *
- * @remarks This is based on the item's `kind`. See {@link doesItemKindRequireOwnDocument}.
+ * @remarks
+ *
+ * This is essentially a wrapper around {@link DocumentationSuiteOptions.hierarchyBoundaries}, but also enforces
+ * system-wide invariants.
+ *
+ * Namely...
+ *
+ * - `Package` items are *always* rendered to their own documents, regardless of the specified boundaries.
+ *
```

Comment:
If there's a blank line in between, these won't be rendered as part of the same list, would they?

Reply:
Actually, they _have_ to be separated by a blank line to be considered part of the same list. Single line breaks are parsed as soft breaks (since devs line break mid sentence, etc. to conform to line length restrictions and such).

https://github.com/microsoft/FluidFramework/wiki/TSDoc-Guidelines#line-breaks

Reply:
Ohhh that's good to know